networks/suzuka/indexer/src/main.rs

```diff
@@ -16,8 +16,8 @@ fn main() -> Result<(), anyhow::Error> {
 		.init();
 
 	let dot_movement = dot_movement::DotMovement::try_from_env()?;
-	let maptos_config = dot_movement
-		.try_get_or_create_config_from_json::<maptos_execution_util::config::Config>()?;
+	let maptos_config =
+		dot_movement.try_get_config_from_json::<maptos_execution_util::config::Config>()?;
```

Comment:
It's not the right correction. If you do that the config is not created if absent. The methos to call is `try_get_or_create_config_from_json` I don't know why it diseapper.

Reply:
This PR is just to ensure `main` will build, but yes if you could add that follow up PR that would be great.